workbench/src/main/menubar.js

```diff
@@ -115,38 +119,44 @@ function openAboutWindow(parentWindow, isDevMode) {
     height: 800,
     frame: true,
     webPreferences: {
-      contextIsolation: false,
-      enableRemoteModule: false,
-      nodeIntegration: true,
+      contextIsolation: true,
+      nodeIntegration: false,
```

Comment:
I do like defining these, but I think this is the default for Electron now. In case we wanted to avoid writing these out in multiple places.

Reply:
Yes you're right, these are the defaults now. Probably makes sense to remove them, at least once we've all noted the change.

Reply:
I've removed these now.